src/components/pages/Wallet.tsx

```diff
@@ -187,7 +186,9 @@ const inviteSectionTextStyle: TextStyle = {
 };
 
 const buttonStyle: ViewStyle = {
-  marginVertical: 8
+  marginVertical: 8,
+  paddingVertical: 16,
+  minWidth: 300
```

Comment:
You can push with this, but why still `300`? If you want it to be full width I think there are better ways than this... I think `width: 100%` should work, but if it doesn't you can put the button in a `View` with `flexDirection: row` and giving the button `flex: 1`. The parent view would automatically fill to full width and the button would then grow to full width of the parent. `300` might look good on your device but isn't flexible to others and makes for inconsistent visuals in the app.

Reply:
If you just don't want to deal with this right now, then I'll make the button component more expressive to our needs later on. The fact you'd like to change the padding for this button and this one only seems to reflect that the button styles are not generalizing to different situations, so having choices for standardized button sizing is probably useful as well.

Reply:
although from the screenshots i cant actually tell what changing the button padding is doing. can you just remove that line?

Reply:
fixed this.